src/common/schema/application.js

```diff
@@ -128,9 +128,19 @@ module.exports = {
       minimum: 0,
       maximum: 9999999.99,
     },
+    states: {
+      oneOf: [
+        {
+          'enum': states
+        }, {
+          type: 'string',
+          pattern: '^$'
+        }
+      ]
+    },
     phone: {
       type: 'string',
-      pattern: '^[0-9]{10}$'
+      pattern: '^[0-9]{10}$|^$'
```

Comment:
The pattern at the API layer should not accept blank.

The changes to `phone` and `state` should likely be reverted in favor of removing them from the object before it is even sent in the server.